tailwind.config.js

```diff
@@ -5,6 +5,7 @@ const tailwindConfig = {
     './pages/**/*.{vue,js,css}',
     './assets/**/*.{vue,js,css}',
     './components/**/*.{vue,js,css}',
+    './content/**/*.md',
```

Comment:
This tells Tailwind to look at our markdown files for Tailwind classes.
Ideally we should transform the files to HTML first: https://tailwindcss.com/docs/content-configuration#transforming-source-files although I'm having trouble with the configuration/ imports. 
This could be left as another task.

Another option is we just use inline styling for image heights and avoid tailwind processing these files as it's not really necessary.

Reply:
Another option is to `safeList` the class we want so it doesn't get purged.